dao-voting/README.md

```diff
@@ -15,7 +15,6 @@ More information can be found in the [specification](./SPECIFICATION.md).
 
 - The smart contract is mostly complete for the basic implementation. There are some issues that need to be worked out but a UI can be started
```

Comment:
Do these issues have... issues? 😄

Reply:
Yes they do. There is a bug, some design considerations and general improvements to be made in future iterations. These aren't necessarily obvious just by glancing at the current [issues](https://github.com/FuelLabs/sway-applications/issues?q=is%3Aopen+is%3Aissue+label%3A%22App%3A+DAO%22) but they do exist.